huaweicloud/services/cbr/resource_huaweicloud_cbr_replicate_backup.go

```diff
@@ -0,0 +1,168 @@
+package cbr
+
+import (
+	"context"
+	"strings"
+
+	"github.com/hashicorp/go-uuid"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+var replicateBackupNonUpdatableParams = []string{
+	"backup_id",
+	"replicate.0.destination_project_id",
```

Comment:
Please confirm whether the hierarchical structure using `.0.` is effective and whether it has the same function as `.*.`.